src/frontends/paddle/tests/test_models/gen_scripts/generate_2in_2out.py

```diff
@@ -14,24 +14,24 @@
 inp_blob1 = np.random.randn(1, 1, 3, 3).astype(np.float32)
 inp_blob2 = np.random.randn(1, 2, 3, 3).astype(np.float32)
 
-x1 = fluid.data(name='inputX1', shape=[1, 1, 3, 3], dtype='float32')
-x2 = fluid.data(name='inputX2', shape=[1, 2, 3, 3], dtype='float32')
+x1 = paddle.static.data(name='inputX1', shape=[1, 1, 3, 3], dtype='float32')
+x2 = paddle.static.data(name='inputX2', shape=[1, 2, 3, 3], dtype='float32')
```

Comment:
I think it makes sense to have tests supporting different versions of pdpd.
Let's just check a version PDPD and use appropriate API.

Reply:
> I think it makes sense to have tests supporting different versions of pdpd. Let's just check a version PDPD and use appropriate API.

You are right, but versions prior to paddle 2.5 do not support python11.

Reply:
I think it is very helpful to test in different paddle version environment. Can you do this to import different api according to the paddle version?

Reply:
> I think it is very helpful to test in different paddle version environment. Can you do this to import different api according to the paddle version?

ok, thanks for your suggestion.